src/uptane_generator/repo_test.cc

```diff
@@ -497,6 +497,126 @@ TEST(uptane_generator, generateCampaigns) {
   EXPECT_EQ(campaigns["campaigns"][0]["metadata"][2]["value"], "20");
 }
 
+/*
+ * Bump the version of the Director Root metadata.
+ */
+TEST(uptane_generator, refreshDirectorRoot) {
```

Comment:
These tests looks really similar.
Would be cool to make most of their body common, with `TEST_P` or one test.

Might be impractical or just as ugly though, no big deal.

Reply:
Yeah, these are pretty repetitive. I haven't gotten into parameterized testing much yet. Would be cool to do, just haven't had the time yet.

Reply:
Ok, in this case it's probably not worth holding up this PR then.